.github/workflows/ci-build.yaml

```diff
@@ -158,6 +159,7 @@ jobs:
       env:
         XLA_FLAGS: "--xla_force_host_platform_device_count=8"
         JAX_ARRAY: 1
+        PY_COLORS: 1
```

Comment:
Can we do this in kokoro too?

Reply:
Not sure. Let's have that discussion offline.